checkout-extension/tsconfig.json.liquid

```diff
@@ -0,0 +1,27 @@
+{%- if flavor contains "preact" -%}
+{
+  // This tsconfig.json file is only needed to inform the IDE.
```

Comment:
We actually should remove this comment. The tsconfig is now required to build preact extensions properly and also to do the type checks.

Can you use the template [here](https://github.com/Shopify/extensions-templates/blob/63a1d061e8799633257d21c845a59b4068648b1c/admin-action/tsconfig.json.liquid#L1)?